cmd/gitops/add/profiles/cmd_test.go

```diff
@@ -0,0 +1,118 @@
+package profiles_test
+
+import (
+	"github.com/go-resty/resty/v2"
+	"github.com/jarcoal/httpmock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/root"
+)
+
+var _ = Describe("Add Profiles", func() {
+	var (
+		cmd *cobra.Command
+	)
+
+	BeforeEach(func() {
+		client := resty.New()
+		httpmock.ActivateNonDefault(client.GetClient())
+		defer httpmock.DeactivateAndReset()
```

Comment:
Doesn't this cause it to be deactivated and reset when it leaves the func() in the `BeforeEach` ?

Reply:
Right, I think this needs to be in an AfterEach, right? 🤔

Reply:
Done!